### PR TITLE
Use app compat drawables.

### PR DESCRIPTION
**Pull request by Petrulak**
Use custom attribute to set drawable.
Old code was crashing on API 19 with combination of a vector drawable.

### CODE REVIEW

File: onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java [hunk 1]
```diff
@@ -107,6 +111,39 @@ private void initializePlaceholderButton(Context context, AttributeSet attrs, in
...
     }
 
+    void initAttrs(Context context, AttributeSet attrs) {
+        if (attrs != null) {
+            TypedArray attributeArray = context.obtainStyledAttributes(attrs, R.styleable.OnusButton);
+
+            Drawable drawableLeft = null;
+            Drawable drawableRight = null;
+            Drawable drawableBottom = null;
+            Drawable drawableTop = null;
```
Comment (JlUgia) [inline onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java]: Space in here.

File: onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java [hunk 2]
```diff
@@ -34,11 +36,13 @@ public OnusButton(Context context) {
 
     public OnusButton(Context context, AttributeSet attrs) {
         this(context, attrs, android.R.attr.buttonStyle);
+        initAttrs(context, attrs);
```
Comment (JlUgia) [inline onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java]: Not needed here.
This calls the `bigger` constructor.

File: onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java [hunk 3]
```diff
@@ -107,6 +111,39 @@ private void initializePlaceholderButton(Context context, AttributeSet attrs, in
...
+
+            Drawable drawableLeft = null;
+            Drawable drawableRight = null;
+            Drawable drawableBottom = null;
+            Drawable drawableTop = null;
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                drawableLeft = attributeArray.getDrawable(R.styleable.OnusButton_drawableLeftCompat);
+                drawableRight = attributeArray.getDrawable(R.styleable.OnusButton_drawableRightCompat);
+                drawableBottom = attributeArray.getDrawable(R.styleable.OnusButton_drawableBottomCompat);
+                drawableTop = attributeArray.getDrawable(R.styleable.OnusButton_drawableTopCompat);
```
Comment (JlUgia) [inline onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java]: Space after this line.

File: onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java (hunk 2 above)
Comment (Petrulak) [inline onusbutton/src/main/java/one/noa/onusbutton/OnusButton.java]: aaaa , yes, your right

**Pull request merged**